deploy/init/01-starboard.ns.yaml

```diff
@@ -1,7 +0,0 @@
-apiVersion: v1
```

Comment:
I didn't check into why it's OK to delete these YAML files, but I imagine you know what you're doing :-)

Reply:
:) Basically it's a duplicate of what we already have in deploy/static/*.yaml